lib/tests/streamlit/git_util_test.py

```diff
@@ -76,3 +76,8 @@ def test_git_repo_valid(self):
             repo = GitRepo(".")
             self.assertTrue(repo.is_valid())
             self.assertEqual((2, 20, 3), repo.git_version)
+
+    def test_gitpython_not_installed(self):
+        with patch.dict("sys.modules", {"git": None}):
+            repo = GitRepo(".")
+            self.assertFalse(repo.is_valid())
```

Comment:
This would return true in our CI pipelines when the `git` module is not set to `None` since we are testing inside a Git repo, or?

Reply:
`repo.is_valid()` returns `False` when basically anything goes on while initializing the object, including
* `git` not being installed
* the system's `git` version being too old
* `gitpython` not being installed
* etc

In this case, we're testing the code path where the import fails